### PR TITLE
Appeals v2 timeline style tweaks

### PR DESCRIPTION
**Pull request by cvalarida**
The original design can be found [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5637).

image

**Note:** There's still some work left to do concerning the text spilling out past the divider lines, but that's an easy fix after we get the tabs PR merged in. Should just be able to take away the `right-margin` from the `.separator` and call it good. :crossed_fingers:

### CODE REVIEW

**Review by gnakm — APPROVED** [review]
Looking g8!

File: src/sass/claims-status.scss [hunk 1]
```diff
@@ -668,7 +678,21 @@ h1:focus {
...
   .appeal-event-date {
-    color: $color-gray-light;
+    color: $color-gray-medium;
+  }
+
+  & + .down-arrow {
+    width: 0px;
+    border-left: 0.6em solid transparent;
+    border-right: 0.6em solid transparent;
+    border-top: 1.2em solid #d6d7d9;
```
Comment (annekainicUSDS) [inline src/sass/claims-status.scss]: Very minor, but is there a variable already for this hex color? Might be `$color-gray-lighter`?

**Review by annekainicUSDS — APPROVED** [review]
LGTM! One minor comment

File: src/sass/claims-status.scss (hunk 1 above)
Comment (cvalarida) [reply src/sass/claims-status.scss]: 😂 
@robbiethegeek Just asked about this too and I made a PR to fix what's already been merged into master. Looped back around to fix this up and saw this comment.

Good call! `$color-gray-lighter` is exactly what I want. 😄

**Pull request merged**